exporter.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import time
+from os import getenv
+
+import os
+
+import urllib3
+from prometheus_client import REGISTRY, start_http_server
+
+from collectors.PyVimServiceCollector import PyVimServiceCollector
+from collectors.SshServiceCollector import SshServiceCollector
+from collectors.EsxiOverallStateCollector import EsxiOnlineStateCollector
+from modules.Exceptions import VCenterException
+
+# init logger once
+logger = logger = logging.getLogger('esxi-exporter')

```

Comment:
Two times logger?

Reply:
Two is better. Just kidding. Thanks for your sharp eyes.

Done.